docs/samples/configuration.md

```diff
@@ -0,0 +1,42 @@
+[
+  {
+    "name": "BlobStorageConnectionString",
+    "value": "DefaultEndpointsProtocol=[https];AccountName=[storageAccountName];AccountKey=[storageAccountKey];EndpointSuffix=[storageAccountEndpointSuffix]",
+    "slotSetting": false
+  },
+  {
+    "name": "BlobStorageContainerName",
+    "value": "",
+    "slotSetting": false
+  },
+  {
+    "name": "CustomerInsightsEntity",
+    "value": "",
+    "slotSetting": false
+  },
+  {
+    "name": "FhirDataConnection:BaseUrl",
+    "value": "https://[FHIRServiceName].azurehealthcareapis.com/",
+    "slotSetting": false
+  },
+  {
+    "name": "FhirDataConnection:ClientId",
+    "value": "",
+    "slotSetting": false
+  },
+  {
+    "name": "FhirDataConnection:ClientSecret",
+    "value": "",
+    "slotSetting": false
+  },
+  {
+    "name": "FhirDataConnection:Tenant",
+    "value": "",
+    "slotSetting": false
+  },
+  {
+    "name": "TimerTrigger",
+    "value": "0 0/5 * * * *",
```

Comment:
This is every 5 minutes. Is that intentional?

Reply:
yea, for testing purposes only so they don't have to wait too long to see it run. I put a comment about this in the setup.md... "You can use this template [configuration.md](./samples/configuration.md) for quick addition of these app settings via the Advanced editor. It includes a sample cron expression to run every 5 minutes for testing purposes."